docs/content/troubleshooting/csi_ug_troubleshooting_node_crash.md

```diff
@@ -0,0 +1,76 @@
+# Recovering a pod volume attachment from a crashed Kubernetes node
+
+This section details a manual operation required to revive Kubernetes pods that reside on a crashed node due to an existing Kubernetes limitation.
+
+## Identifying a crashed node
+**Note:** These procedures are applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.
+
+When a worker node shuts down or crashes, all pods in a StatefulSet that reside on it become unavailable. In these scenarios, the node status is _NotReady_, and the pod status appears as _Terminating_.
+
+For example:
+
+```screen
+$> kubectl get nodes
+NAME STATUS ROLES AGE VERSION
+k8s-master Ready master 6d <your k8s version>
+k8s-node1 Ready <none> 6d <your k8s version>
+k8s-node3 NotReady <none> 6d <your k8s version>
+
+$> kubectl get pods --all-namespaces -o wide | grep default
+default sanity-statefulset-0 1/1 Terminating 0 19m 10.244.2.37 k8s-node3
+```
+
+## Recovering a crashed node
+
+**Attention:** In order to avoid data loss, before continuing, verify that there are no pods connected to this volume.
+
+Follow the following procedure to recover from a crashed node (see a [full example](#full_example) at the end of the procedure):
+
+1.  Find for the `volumeattachment` of the created pod:
```

Comment:
consider removing the word "for" from the sentence

Reply:
done in 3.9.0

Reply:
Thanks.

Reply:
1.9.0*

Reply:
@samishekh please review the change, and if you approve, please resolve this conversation (otherwise continue it).
thanks